.slackbot.yml

```diff
@@ -0,0 +1,15 @@
+releaseSchedule:
+  dbagnell: 2/1/2019
```

Comment:
@OmarShehata Dan will be after Sean.  Gabby is up for the next release.

Reply:
Looking at the blog posts, we skipped both Gary and Dan.

Reply:
Yeah, but for simplicity we decided we were just going to restart the cycle

Reply:
Should be correct now @hpinkos would you like to merge?